packages/report-server/src/__tests__/reportBuilder/transform/parser.test.ts

```diff
@@ -62,11 +67,11 @@ describe('parser', () => {
     ]);
   });
 
-  it('sort supports row parser lookups', () => {
+  it('sortRows supports row parser lookups', () => {
     const transform = buildTransform([
       {
-        transform: 'sort',
-        by: '$row.BCD1',
+        transform: 'sortRows',
+        by: '=$BCD1',
```

Comment:
Oh I forgot to mention this point - I am a little nervous of people getting confused about when to use the `=` (i.e. `Total` and `=$BCD1` vs. `'Total'` and `$BCD1`). I had thought it was pretty clean because it's just `=` required on the LHS, not required on RHS, but then I noticed this example where we (very sensibly!) default to using literals the RHS rather than automatically evaluating. One option is to use `=` everywhere, like excel does, but it's a big amount of clutter to add to the very common use case. Any thoughts?

Reply:
Yeah I'm a bit unsure too...

Given that `=` isn't too hard to type out, part of me in inclined to go with it everywhere...

Another thought is that if we get a UI around this, we can have the input field itself be toggleable between expression and non-expression, and then maybe we can add sensible defaults there?

Reply:
Good call on the UI (side note, more and more I'm thinking that even when the UI exists, power users will want to write these freehand, because it will just be more fluid if you're creating a bunch)

I'm somewhat inclined to use `=` everywhere too...

Reply:
Should we trial `=` everywhere and see how it goes? We can always go back if it's annoying?

Reply:
See what Gerry thinks in your session today ;-)

Reply:
Okay, interestingly @gerrybes did actually seem to find the `=` in one place but not the other to be a bit confusing! So I'm pretty keen to try `=` is always required for expressions 👍

One other thing he noted was:
`=$a + $b` is a tad confusing, at least in comparison to `=($a + $b)`. I can definitely see his point here, however I feel forcing people to wrap all their expressions in quotes is a bit too verbose. Since expression-parser will allow you to wrap or not wrap, we can leave this is an optional stylistic thing?